src/main/java/org/example/homework10/Main.java

```diff
@@ -0,0 +1,108 @@
+        songListBeatles.addSong(songTwo);
+        songListBeatles.addSong(songThree);
+        System.out.println("Adding song that already exist in the album. Result: " + songListBeatles.addSong(songFour));
+        System.out.println("Find a song by the name: " + songTwo.getSongName() + " " + songListBeatles.findSongByTitle("girl"));
+        System.out.println("Find a song by the number: " + songThree.getSongId() + " " + songListBeatles.findSongByTrackNumber(3));
+        System.out.println("Find a song number that doesn't exist in the album. Result: " + songListBeatles.findSongByTrackNumber(4));
+
+        System.out.println(("*").repeat(80));
+
+        System.out.println("**Employee Processing with Local Classes and Anonymous Sorting task**");
+        Employee e1 = new Employee("Minnie", "Mouse", "2000-11-27");
+        Employee e2 = new Employee("Mickie", "Mouse", "2009-01-07");
+        Employee e3 = new Employee("Daffy", "Duck", "1999-12-30");
+        Employee e4 = new Employee("Daisy", "Duck", "2015-12-01");
+        Employee e5 = new Employee("Goofy", "Dog", "1824-05-30");
+
+        List<Employee> list = new ArrayList<>(Arrays.asList(e1, e2, e3, e4, e5));
+        EmployeeData employeeData = new EmployeeData();
+        employeeData.calculated(list);
+
+        employeeData.printOrderedList("Name");
+        System.out.println();
+        employeeData.printOrderedList("year");
+
+        System.out.println(("*").repeat(80));
+
+        System.out.println("**Split the sentence - lambda**");
+        LambdaClass.splitAndPrint("I will return to Shangri-La");
+    }
+
+    public static class EmployeeData {
+        String fullName;
+        Integer years;
+        ArrayList<EmployeeData> calculatedEmployeesList;
+
+        public EmployeeData() {
+            this.calculatedEmployeesList = new ArrayList<>();
+        }
+
+        public EmployeeData(String fullName, Integer years) {
+            this.years = years;
+            this.fullName = fullName;
+        }
+
+        @Override
+        public String toString() {
+            return "Calculated Employee Data{" +
+                    "fullName: '" + fullName + '\'' +
+                    ", worked years = " + years +
+                    '}' + "\n";
+        }
+
+        public void calculated(List<Employee> employee) {
+            for (Employee value : employee) {
+                String firstName = value.firstName();
+                String lastName = value.lastName();
+                String dataYears = value.hireDate();
+                LocalDate dateOfStart = LocalDate.parse(dataYears);
+
+                fullName = firstName + " " + lastName;
+                years = LocalDate.now().getYear() - dateOfStart.getYear();
+                calculatedEmployeesList.add(new EmployeeData(fullName, years));
+            }
+        }
+
+        public void printOrderedList(String orderBy) {
+            if (orderBy.equalsIgnoreCase("year")) {
```

Comment:
You did not implement this via Local Class. The idea was to create local class, e.g. `MyEmployee` inside the `printOrderedList` method